apps/aechannel/src/aesc_offchain_tx.erl

```diff
@@ -242,7 +242,13 @@ version(#channel_offchain_tx{updates = Updates}) ->
 -spec valid_at_protocol(aec_hard_forks:protocol_vsn(), tx()) -> boolean().
 valid_at_protocol(Protocol, Tx) ->
     case version(Tx) of
-        ?INITIAL_VSN -> true;
+        ?INITIAL_VSN when Protocol >= ?LIMA_PROTOCOL_VSN -> false;
```

Comment:
This line is not essentially part of this PT but while we are here, we can also forbid old style off-chain transactions making it on-chain. This will allow us not to worry of off-chain updates making it on-chain ever again. Post Lima that is :)